WordPress/Classes/Utility/JetpackREST.m

```diff
@@ -0,0 +1,47 @@
+#import "JetpackREST.h"
+#import "AccountService.h"
+#import "BlogService.h"
+#import "ContextManager.h"
+
+static NSString * const JetpackRESTEnabledKey = @"JetpackRESTEnabled";
+
+@implementation JetpackREST
+
++ (BOOL)enabled
+{
+    return [[NSUserDefaults standardUserDefaults] boolForKey:JetpackRESTEnabledKey];
+}
+
++ (void)setEnabled:(BOOL)enabled withCompletion:(void (^)())completion
+{
+    NSUserDefaults *defaults = [NSUserDefaults standardUserDefaults];
```

Comment:
I am wondering if it would make sense to check if the current flag is the same one we're trying to set and just `return` in that case. Since we are changing this flag mostly from the UI, it might be an overkill too. Your call :)